kolibri/plugins/learn/assets/src/views/content-card/index.vue

```diff
@@ -83,6 +85,20 @@
       mastered() {
         return this.progress === 1;
       },
+      cardHeight() {
```

Comment:
@indirectlylit not sure how I feel about this approach. 

Using responsive element now to get a pixel value inside of content-card, basically making all the size changes rely on the width.

Initially, I wanted to leave content-card alone and have all the changes happen in the grid. Any thoughts on this?